common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -355,12 +388,14 @@ class CheckboxProblemTypeTest(ProblemTypeTestBase, ProblemTypeTestMixin):
     """
     problem_name = 'CHECKBOX TEST PROBLEM'
     problem_type = 'checkbox'
+    partially_correct = True
 
     factory = ChoiceResponseXMLFactory()
 
     factory_kwargs = {
-        'question_text': 'The correct answer is Choice 0 and Choice 2',
+        'question_text': 'The correct answer is Choice 0 and Choice 2, Choice 1 and Choice 3 together are incorrect.',
         'choice_type': 'checkbox',
+        'credit_type': 'edc',
```

Comment:
It is a form of partial credit for checkbox problem types.

http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/checkbox.html?highlight=edc#id27